bound/bound_test.go

```diff
@@ -75,3 +77,45 @@ func TestIsValid(t *testing.T) {
 		t.Error("b4 is invalid")
 	}
 }
+
+func TestUnion(t *testing.T) {
+	b0 := Bound{Min: 0.0, Max: 5.0}
+	b1 := Bound{Min: 3.0, Max: 4.0}
+	b2 := Bound{Min: -1.0, Max: 1.0}
+	b3 := Bound{Min: 1.0, Max: 2.0}
+	b4 := Bound{Min: 1.1, Max: 2.0}
+
+	ret := Union(b0)
```

Comment:
@kortschak 

Shall I update this one also into `union := Union(b0)`?
My intent was the `return value VS expected value`  for testing.

Reply:
You can leave the tests. We usually use `got` for test results, but this will be rewritten to use tables.

Reply:
Okay!